Assets/Scripts/Utilities/AudioZone.cs

```diff
@@ -0,0 +1,31 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class AudioZone : MonoBehaviour 
+{
+	[SerializeField] AudioSource sourceData = null;
+	AudioSource playingClip = null;
+
+	void OnTriggerEnter( Collider col )
+	{
+		if ( col.GetComponent<PlayerActor>() )
```

Comment:
Yeah, we can talk about how best to do that. At the least I'd like to not tie it down to this particular game so that we can reuse the sound manager in the future without having to make any modifications.